lua/lang/ruby.lua

```diff
@@ -41,24 +41,26 @@ M.lint = function()
 end
 
 M.lsp = function()
-  if require("lv-utils").check_lsp_client_active "solargraph" then
```

Comment:
this line was to avoid spawning a new `solargraph` lsp for each nvim instance, so ...